assets/shared/helpers/data.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Return keys of a key-value map where their value is true.
+ *
+ * @param {Object} map Data.
+ * @return {string[]} Selected keys.
+ */
+export function getSelectedKeys( map ) {
+	return Object.entries( map ).reduce( ( m, [ key, value ] ) => {
```

Comment:
I thought a little more about this function. And WDYT of something like `Object.keys( map ).filter( key => map[ key ] )`. This solution would be also a O(n) over the keys.

Reply:
Nice! 93b3c1f